File: cases/taint/c/expression-negative/expression_negative.c
int dfb_source(void) { // DFB-SOURCE: expression-negative-input
    return 1;
}

void dfb_sink(int value) {} // DFB-SINK: expression-negative-sink

void run(void) {
    int value = dfb_source();
    int computed = (value * 3) + 7; // DFB-WITNESS: expression-negative-computed
    dfb_sink(7);
}

File: cases/taint/rust/expression-negative/expression_negative.rs
fn dfb_source() -> i32 { // DFB-SOURCE: expression-negative-input
    1
}

fn dfb_sink(value: i32) {} // DFB-SINK: expression-negative-sink

fn run() {
    let value = dfb_source();
    let computed = (value * 3) + 7; // DFB-WITNESS: expression-negative-computed
    dfb_sink(7);
}
